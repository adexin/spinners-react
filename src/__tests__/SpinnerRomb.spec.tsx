import { render } from '@testing-library/react';
import React from 'react';

import { SpinnerRomb } from '../SpinnerRomb';

describe('SpinnerRomb', () => {
  it('exists for backward compatibility', () => {
    const { container } = render(
      <SpinnerRomb color="#fff" size={100} speed={10} thickness={50} />,
    );

    expect(container).toMatchSnapshot();
  });
});
