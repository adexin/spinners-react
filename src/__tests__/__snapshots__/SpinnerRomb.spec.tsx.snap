// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpinnerRomb exists for backward compatibility 1`] = `
<div>
  <svg
    fill="currentColor"
    style="color: rgb(255, 255, 255); overflow: visible; width: 100px;"
    viewBox="0 0 96 96"
  >
    <circle
      cx="3"
      cy="48"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="18"
      cy="33"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="18"
      cy="48"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="18"
      cy="63"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="33"
      cy="48"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="33"
      cy="18"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="33"
      cy="33"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="33"
      cy="63"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="33"
      cy="78"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="48"
      cy="3"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="48"
      cy="18"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="48"
      cy="33"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="48"
      cy="48"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="48"
      cy="63"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="48"
      cy="78"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="48"
      cy="93"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="63"
      cy="18"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="63"
      cy="33"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="63"
      cy="48"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="63"
      cy="63"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="63"
      cy="78"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="78"
      cy="33"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="78"
      cy="48"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="78"
      cy="63"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <circle
      cx="93"
      cy="48"
      fill="rgba(0,0,0,0.44)"
      r="1.25"
    />
    <g
      style="animation: spinners-react-diamond 14s steps(2, end) infinite;"
    >
      <circle
        cx="3"
        cy="48"
        r="1.75"
      />
      <circle
        cx="18"
        cy="33"
        r="1.75"
      />
      <circle
        cx="18"
        cy="48"
        r="1.75"
      />
      <circle
        cx="18"
        cy="63"
        r="1.75"
      />
      <circle
        cx="33"
        cy="48"
        r="1.75"
      />
    </g>
  </svg>
</div>
`;
